source/chapters/appendix.rst

```diff
@@ -305,19 +305,16 @@ Option                  Description
                         for user settings files such as the library
                         database and preferences configuration file.
 ----------------------  -------------------------------------------------
---controllerDebug       Causes Mixxx to display/log all controller data
-                        it sends and receives, and script functions it
-                        loads.
+--controllerDebug       Display/log all controller data Mixxx sends and
+                        receives as well as script functions it loads.
 ----------------------  -------------------------------------------------
---developer             Enables developer-mode. Includes extra log info,
-                        stats on performance, a Developer tools menu,
-                        extra developer skins, and provides useful
-                        information for skin writers in widget tooltips
-                        and logs .
+--developer             Enable developer-mode. Includes extra logs, stats
+                        on performance, a Developer tools menu and extra
+                        developer skins. Also provides useful information
```

Comment:
there are no "developer only" skins.. ;)

Reply:
Speaking of which, can we delete https://github.com/mixxxdj/developer_skins already?

Reply:
Enable developer-mode. Includes extra logs, stats on performance and the Developer tools menu.
Also, logs and tooltips provide extra info for skin developers.

Reply:
> Speaking of which, can we delete https://github.com/mixxxdj/developer_skins already?

doesn't hurt anyone, does it? Let's keep it until someone (me?) did the next rev of the skin creation docu.
Even then, should we shrink-pack it and put it somewhere for documentation?

Reply:
Documenting code that is years out of date would be a waste of time that could only create confusion

Reply:
We don't keep commented code around, so why should we keep that repository?

Reply:
We can archive the repo for now (that makes it read-only), and delete when we're sure it contains nothing worth keeping.

Reply:
Adjusted the docs :)

Reply:
> We can archive the repo for now (that makes it read-only), and delete when we're sure it contains nothing worth keeping.

Jup. I'll take a look when 2.3 is out the door.